### DISCUSSION TITLE
feature request for ortoni report

### DISCUSSION THREAD

**Discussion by rahulbendale03** [ortoniKC/ortoni-report#14] [Ideas]
Can we change the report name which is saved in the project and the tab name which opens in the browser from config file?

Current report name: ortoni-report.html
Current tab name:  Ortoni Playwright Test Report

**Comment by rahulbendale03** [comment]
@ortoniKC Could you please check.

**Reply by ortoniKC** [reply]
I'll be providing an update in 2 days.

**Reply by rahulbendale03** [reply]
Thanks

**Reply by rahulbendale03** [reply]
Also, can we generate this report in folder instead of in root directory.

**Reply by ortoniKC** [reply]
File name can be configure now, however the report folder will take sometime. 
When we use the ortoni-report CLI it will bundle the report in a ortoni-report folder.

**Reply by rahulbendale03** [reply]
Ok Thanks a lot

**Comment by rahulbendale03** [comment]
Hi @ortoniKC I am getting below error for command npx ortoni-report gr -f Automation-Execution-Report.html
Could you please help.

![2024-08-06_13-57-42](https://github.com/user-attachments/assets/cb51315b-1b82-4926-ac2f-65af435208a0)

**Reply by ortoniKC** [reply]
It's because the project name has space,
As per node js configuration you should have space at the root level. 
Please remove the space and check.

**Reply by rahulbendale03** [reply]
Thanks. Its working now.

**Comment by rahulbendale03** [comment]
Hi @ortoniKC 

Can we insert screenshot in report for each step. Currently screenshot is getting inserted only for last step.

**Reply by ortoniKC** [reply]
Steps level screenshots are added as pagination in the version 2.0.3
